tests/shell/shell_recorder/globalplugins.esr

```diff
@@ -1,32 +0,0 @@
-Mountpoint: system/
-DiffType: File
-
-
-< set system/elektra/globalplugins ""
```

Comment:
Seems like on your system the tracer and timeofday plugin are globally mounted.

`kdb rm -r system/elektra/globalplugins/ ` should do the trick.

We could use `kdb rm -r system/elektra/globalplugins/ ` (and restore afterwards) as workaround as long as we have no way to unmount global plugins?